guides/shipments/overview-of-shipments.md

```diff
@@ -0,0 +1,463 @@
+
+|S. Category / S. Method|DHL|FedEx|USPS|
+|---:|---:|---:|---:|
+|Light|Per Item ($5)|Flat Rate ($10)|Per Item ($8)|
+|Regular|Per Item ($5)|Per Item ($2)|Per Item ($8)|
+|Heavy|Per Item ($50)|Flexi Rate($20,$15)|Per Item ($20)|
+
+## UI
+
+### What the Customer Sees
+
+In the standard system, there is no mention of shipping until the checkout phase.
+
+After entering a shipping address, the system displays the available shipping options and their costs for each shipment in the order. Only the shipping options whose zones include the _shipping_ address are presented.
+
+The customer must choose a shipping method for each shipment before proceeding to the next stage. At the confirmation step, the shipping cost will be shown and included in the order's total.
+
+**Note:*- *You can enable collection of extra _shipping instructions_ by setting the option `Spree::Config.shipping_instructions` to `true`. This is set to `false` by default. See [Shipping Instructions](#shipping-instructions) below.*
+
+### What the Order's Administrator Sees
+
+**Shipment*- objects are created during checkout for an order. Initially each records just the shipping method and the order it applies to. The administrator can update the record with the actual shipping cost and a tracking code, and may also (once only) confirm the dispatch. This confirmation causes a shipping date to be set as the time of confirmation.
+
+## Advanced Shipping Methods
+
+Solidus comes with a set of calculators that should fit most of the shipping situations that may arise. If the calculators that come with Solidus are not enough for your needs, you might want to use an extension - if one exists to meet your needs - or create a custom one.
+
+### Extensions
+
+There are a few Solidus extensions which provide additional shipping capabilities. See the [Solidus Extension List](http://extensions.solidus.io/) for the latest information.
+
+### Writing Your Own
+
+For more detailed information, check out the section on [Calculators](calculators).
+
+Your calculator should accept an array of `LineItem` objects and return a cost. It can look at any reachable data, but typically uses the address, the order and the information from variants which are contained in the line_items.
+
+### Product & Variant Configuration
+
+Store administrators can assign products to specific Shipping Categories or include extra information in variants to enable custom calculators to determine results. Weight and dimension information can also be used in the calculator.
+
+## Shipping Instructions
+
+The option `Spree::Config[:shipping_instructions]` controls collection of additional shipping instructions. This is turned off (set to `false`) by default. If an order has any shipping instructions attached, they will be shown in an order's shipment admin page and can also be edited at that stage. Observe that instructions are currently attached to the _order_ and not to actual _shipments_.
+
+## The Active Shipping Extension
```

Comment:
This explanation is very good, but in my opinion it's too much accurate for something which is not included in core. Also, it's very easy that it goes out of date here. I think this has to be moved in the Active Shipping Extension. The link provided in the first section of this guide could be enough.

Reply:
I'm creating a separate article for the `solidus_active_shipping` gem, but I can see it belonging either as part of that repository or in an "Supported extensions" guide in the future.